test/assets/module/LaminasTestApiToolsDbApi/src/LaminasTestApiToolsDbApi/V1/Rest/Artist/ArtistCollection.php

```
@@ -0,0 +1,8 @@
+<?php
+namespace LaminasTestApiToolsDbApi\V1\Rest\Artist;
+
+use Laminas\Paginator\Paginator;
+
+class ArtistCollection extends Paginator
```

PHPStan (2.2.7)

[error] Class LaminasTestApiToolsDbApi\V1\Rest\Artist\ArtistCollection extends `@final` class Laminas\Paginator\Paginator.


(class.extendsFinalByPhpDoc)